modules/schematics/src/reducer/index.ts

```diff
@@ -60,9 +60,22 @@ export default function (options: ReducerOptions): Rule {
     ]);
 
     return chain([
-      branchAndMerge(chain([addReducerToState(options)])),
       branchAndMerge(
-        chain([addReducerImportToNgModule(options), mergeWith(templateSource)])
+        chain([
+          addReducerToState({
+            ...options,
+            plural: options.feature && options.entity,
```

Comment:
This was related to the comment below - a way to conditionally handle plurals when including entity generation. It will be reverted with the next update.